www/index.html

```diff
@@ -26,22 +26,25 @@
 <div id="platform-instructions-unix" class="instructions" style="display: none;">
   <p>Run the following in your terminal, then follow the onscreen instructions.</p>
   <pre>curl https://sh.rustup.rs -sSf | sh</pre>
+  <p class="other-platforms-help">You appear to be running Unix. Click <a class="default-platform-button" href="#">here</a> to see the installer for all supported platforms.</p>
```

Comment:
Could we rephrase these links to avoid the 'click here' anti-pattern, e.g., make the link text "view all supported installers" or something like that

Reply:
What about : "You appear to be running Unix. If not, <a>display all supported installers</a>."

Reply:
Sounds good to me!